.github/actions/cancel-workflow-run/action.yml

```
@@ -0,0 +1,71 @@
+          });
+          
+          // Find current job details
+          const currentJobDetails = jobs.find(job => job.name === currentJob);
+          
+          // Check if there are other running jobs
+          const otherRunningJobs = jobs.filter(job => 
+            job.status === 'in_progress' && job.id !== currentJobDetails?.id
+          );
+          
+          console.log(`Found ${otherRunningJobs.length} other running jobs`);
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)